docs/administration/datagrid/fields.md

```diff
@@ -0,0 +1,83 @@
+# Fields
+
+Fields are the building blocks of a datagrid. They are used to define the columns of the datagrid and to specify how the data should be displayed.
+
+## Configuration
+
+Fields are defined by the `Datagrid` class using the `add(string $name, array $options)`, `update(string $name, array $options)` and `remove(string $name)` methods. The `$name` parameter is the name of the field and the `$options` parameter is an array of options that define the behavior of the field.
+
+In case you are using `OrmAdapter`, you can use dot notation to access nested properties of the entity. Here are some examples of how to define field names:
+
+- `$datagrid->add('domainId')` - This will fetch the `domainId` property of the entity.
+- `$datagrid->add('currency.code')` - This will join the `currency` relation and fetch the `code` property of the `Currency` entity.
+- `$datagrid->add('status.name')` - This will join the `status` relation and look for the `name` property of the `Status` entity. In this case `Status` is a translatable entity and the `name` property is a translatable field. The datagrid will automatically fetch the correct translation based on the current locale.
+- `$datagrid->add('currency')` - This will fetch the `Currency` entity. In this case you need to define `template`, `transform` or `visible` to `false` option to tell datagrid how to display this object.
+
+### Options
+
+The following options are available for fields:
+
+- `label` - The label of the field. It is displayed in the header of the column.
+- `visible` - If set to `true`, the field is displayed in the datagrid. If set to `false`, the field is hidden, but fetching the data is still performed.
+- `sortable` - If set to `true`, the field can be sorted by clicking on the column header.
+- `virtual` - If set to `true`, the field is not fetched from the data source.
+- `help` - The help text that is displayed next to label in column header.
```

Comment:
help is not rendered if the field is not sortable or is virtual

Reply:
I will look at it

Reply:
Fixed. I misplaced it in template. Now it's working, but that code is placed twice due to weird CSS computing of arrows :D I can't wait for our new administration :D